src/model.jl

```diff
@@ -265,3 +265,8 @@ macro objective(model, sense, expr)
         Parametron.setobjective!($model, $sense, Parametron.@expression $expr)
     end |> esc
 end
+
+@static if VERSION >= v"0.7-"
+    # Support constructs like value.(model, x)
+    Broadcast.broadcastable(m::Model) = Ref(m)
```

Comment:
Sure, we can do this. I'm still annoyed that you have to do this for all non-iterable types now.

Reply:
To clarify, no action needed, just wanted to vent 🙂.